packages/core/src/services/app/user.ts

```diff
@@ -554,6 +556,13 @@ export default class UserService {
         if (updated[0] === 0) {
             throw new BaseError('UPDATE_FAILED', 'user was not updated!');
         }
+
+        LogService.saveLog(
+            user.address,
```

Comment:
you could use updated[0].id I think. THis goes together with my comment on "why not accepting just id?"